src/sagas/repository.ts

```diff
@@ -551,15 +551,22 @@ export function* getDatasetById({ payload }: any): any {
 }
 
 export function* getDatasetPolicy({ payload }: any): any {
-  const datasetId = payload;
+  const { datasetId, options } = payload;
   try {
     const response = yield call(authGet, `/api/repository/v1/datasets/${datasetId}/policies`);
     yield put({
       type: ActionTypes.GET_DATASET_POLICY_SUCCESS,
       policy: response,
     });
   } catch (err) {
-    showNotification(err);
+    if (options?.suppressErrorNotification) {
+      yield put({
+        type: ActionTypes.GET_DATASET_POLICY_SUCCESS,
+        policy: { data: { policies: [{ name: 'ERROR', members: [] }] } },
```

Comment:
I'm not sure if this is "good enough" or if I should add another return field for this one case. I didn't see another field for returning out of band errors like this to the caller.